src/lib/components/ServerId.svelte

```diff
@@ -2,13 +2,37 @@
 	import MenuButton from './MenuButton.svelte';
 	import Icons from '$lib/icons/MenuIcons.json';
 	import { serverId } from '$lib/stores/stateStore';
+	import { fade } from 'svelte/transition';
+
+	let isClicked = false;
+
+	function copyUrl() {
+		let url: string = '64.227.121.226:3000/';
+		url += $serverId;
+
+		navigator.clipboard.writeText(url);
+
+		isClicked = true;
+
+		setTimeout(() => {
+			isClicked = false;
+		}, 1000);
+	}
 </script>
 
 <!-- Simple box where the user can se the server ID -->
-<!-- TODO: Copy the url, when user clicks -->
+{#if isClicked}
+	<div
+		transition:fade={{ delay: 250, duration: 300 }}
+		class="fixed top-0 right-0 flex items-center h-8 px-6 py-8 m-4 font-bold bg-orange-500 rounded-md mt-36"
```

Comment:
```suggestion
		class="fixed top-0 right-0 flex items-center h-8 px-6 py-8 m-4 font-bold bg-green-500 rounded-md mt-36"
```
orange skriger ikke umiddelbart "success"